src/Filters/molecules/FilterDropdown.tsx

```diff
@@ -40,19 +40,6 @@ const OptionList = styled.div<{moreItem?: boolean}>`
   position: relative;
   overflow-y: auto;
   margin-right: 2px;
-  
-  /* Firefox supports the scrollbar-width property */
-  scrollbar-width: thin;
-
-  /* Customize the scrollbar width for Chrome and Safari */
-  ::-webkit-scrollbar {
-    width: 2px;
```

Comment:
If you remove this css then default width for scroll bar will be applied automatically, which doesn't looks good in case of dropdown filters, IMO. 
Have you tried increasing the width from 2px to 10px, 12px? How it looks in that case?

Reply:
Sir, I tried that, but Suzuki san agreed for default width for scrollbar, hence I kept default width. Please check the comment from suzuki san in the provided query.
https://www.bugherd.com/projects/263632/tasks/461

And also let me know if I should keep the earlier css by simply increasing the width from 2px to 10px.

Reply:
If it is the requirement then i am ok with the change.

Reply:
Sure, Thanks @fti-bcarpenter sir